python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion.py

```diff
@@ -60,7 +60,7 @@ def from_dict(cls, settings: Dict[str, str]) -> "OpenAIChatCompletion":
         """
 
         return OpenAIChatCompletion(
-            model_id=settings["model_id"],
+            model_id=settings["ai_model_id"],
```

Comment:
Would it be better to standardize `ai_model_id` across all code instead of having a mix and match or `ai_model_id` and `model_id`?

Reply:
agreed, bit bigger breaking change but much easier to read and understand what's going on.